docs/getting-started.md

```diff
@@ -0,0 +1,98 @@
+- Addresses of DNS servers, NTP servers, and the proxy server (if necessary) 
+
+## 4. Deploy the first cluster node. 
+
+Deployment involves installing the operating system and other components on the host, and then rebooting once installation is completed. Deploying the first node creates the cluster, and the first node is assigned the management node by default. 
+    
+During installation, you must configure node settings, define the cluster management address (VIP) and the cluster token, and specify other information. If necessary, you can configure more settings using a [Harvester configuration](../install/harvester-configuration/) file. 
+
+Once installation is completed, the node restarts and then the Harvester console appears. The console displays information about the cluster (management URL and status) and the node (hostname, IP address, and status). After the cluster is initialized and all services start running, the cluster status changes to **Ready**. 
+
+## 5. Configure a strong password for the default `admin` user on the Harvester UI. 
+
+Once the cluster status changes to **Ready**, you can access the [Harvester UI](../authentication) using the management URL displayed on the console. 
+
+## 6. Deploy the other nodes and join them to the cluster. 
+
+Deployment involves installing the operating system and other components on the host, and then rebooting once installation is completed. All other nodes join the cluster that was created when the first node was deployed. 
+
+During installation, you must configure node settings, and specify the cluster management address (virtual IP) and the cluster token that you defined previously. If necessary, you can configure more settings using a [Harvester configuration](../install/harvester-configuration/) file. 
+
+When the cluster has three or more nodes, the two nodes added after the first node are automatically promoted to management nodes to form a high-availability (HA) cluster. 
+
+## 7. Create a cluster network and a VM network.
+
+Networking in Harvester involves three major concepts:
+
+- [**Cluster network**](../networking/index#cluster-network): Traffic-isolated forwarding path for transmission of network traffic in the Harvester cluster. 
+    
+    During deployment, a cluster network called the [*management network*](../networking/index#built-in-cluster-network) is created for intra-cluster communications. Harvester allows you to create [custom cluster networks](../networking/index#custom-cluster-network) that can be dedicated to VM traffic and that allow VMs to be accessed from both internal and external networks.
+
+- [**Network configuration**](../networking/index#cluster-network): Definition of how cluster nodes connect to a specific cluster network. 
+    
+    Each network configuration corresponds to a set of nodes with uniform network specifications. Only nodes that are covered by the network configuration can access the associated cluster network.
```

Comment:
People are usually confused by the concept of cluster networks and network configuration. However, we can mention that this provides flexibility in configuration when the cluster consists of machines of various specs, especially the network interface names.

Reply:
Thank you. PTAL at the updated text.